shr-tag-pre-highlight.el

```diff
@@ -193,6 +193,8 @@ Adapted from `org-src--get-lang-mode'."
 (defun shr-tag-pre-highlight-fontify (code mode)
   "Fontify CODE with Major MODE."
   (with-temp-buffer
+    (when (boundp 'untrusted-content)
+      (set 'untrusted-content t))
```

Comment:
note: I'm using `set` to avoid any warnings on older Emacs versions where `untrusted-content` isn't defined.

also note: `untrusted-content` is permanently local.